proxy/examplelogs/logs.txt

```diff
@@ -0,0 +1,6 @@
+This text document helps define each example log file. These files help users demo as well as test and develop with FaradayRF software without the need for hardware.
+---
+
+Log: 4-20-2017_1.db
```

Comment:
I recommend standardizing on YYYY-MM-DD-##.db as it sorts better alphabetically.